### PR TITLE
matplotlib version issue

### PR DESCRIPTION
**Pull request by dzelge**
When accepting this PR:

- the matplotlib maximal dependency restriction '<3.3.0' will have been removed
- The conda env will build
- the op animate will work again

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -13,6 +13,8 @@
   - Parameter `crs` of function `cate.ops.io.read_geo_data_frame` 
 * Fixed bug with user preferences not being saved correctly.
   ([#146](https://github.com/CCI-Tools/cate-app/issues/146))
+* Fixed environment not building with matplotlib version `<3.3.0` ([#929](https://github.com/CCI-Tools/cate/issues/929))
```
Comment (forman) [inline CHANGES.md:16]:
```suggestion
* Fixed environment not building with matplotlib version `<3.3.0`. (#929)
```

File: CHANGES.md [hunk 2]
```diff
@@ -13,6 +13,8 @@
...
+* Fixed the operator `animate`not working with the current xarray version (0.18)
```
Comment (forman) [inline CHANGES.md:17]:
```suggestion
* Fixed the operation `animate_map` that stopped working with xarray version 0.18.0.
```

**Review by forman — CHANGES_REQUESTED** [review]
Good! Changes required just in CHANGES.md.

**Review by forman — APPROVED** [review]

**Review by TonioF — CHANGES_REQUESTED** [review]
Almost perfect - but could you please also add this fix to cate.ops.plot, where one test fails due to the exact same problem?

**Review by TonioF — APPROVED** [review]
Now you could adjust the entry in the change log ... but I'm happy to accept it as it is.

**Review by forman — APPROVED** [review]

**Pull request merged**